subprojects/plugins/src/integTest/groovy/org/gradle/java/compile/AbstractIncrementalCompileIntegrationTest.groovy

```diff
@@ -34,34 +37,35 @@ abstract class AbstractIncrementalCompileIntegrationTest extends AbstractIntegra
         file("src/main/${language.name}/Test.${language.name}") << 'public class Test{}'
         buildFile << """
             apply plugin: '${language.name}'
+            java.toolchain.languageVersion = JavaLanguageVersion.of(11)
```

Comment:
🤔 I think we lose a little coverage here and make this more elaborate than it needs to be.

Instead of using a toolchain here, could we change this to set source compatibility to `Jvm.current() - 1` and then change it to `Jvm.current()`?

I think the only thing we care about is changing these properties does something.